rust/connlib/tunnel/src/gateway/flow_tracker.rs

```diff
@@ -33,6 +33,18 @@ pub struct FlowTracker {
     created_at_utc: DateTime<Utc>,
 }
 
+/// Additional properties we track for a client.
+#[derive(Debug, Clone, Default)]
+pub struct ClientProperties {
+    pub version: String,
+    pub device_serial: Option<String>,
+    pub device_uuid: Option<String>,
+    pub identifier_for_vendor: Option<String>,
+    pub firebase_installation_id: Option<String>,
+    pub identity_name: String,
+    pub actor_email: String,
```

Comment:
These will be optional

```suggestion
    pub identity_name: Option<String>,
    pub actor_email: Option<String>,
```

Reply:
Hmmm you could actually make `actor_email` required but will need to use a placeholder for today's service account actors.